sources/targets/Stride.Core.targets

```diff
@@ -46,7 +46,7 @@
         Condition="'$(StrideEnableCodeAnalysis)' != 'true'">
     <ItemGroup>
       <!-- We want to include Stride analyzers by default -->
-      <AnalyzersNotToRemove Include="@(Analyzer)" Condition="$([System.String]::Copy(%(FullPath)).Contains('Stride'))" />
+      <AnalyzersNotToRemove Include="@(Analyzer)" Condition="$(FullPath.Contains('Stride'))" />
```

Comment:
It looks like this line is the only thing that is needed to fix it. I'm not sure why it is triggering now, since that wasn't part of the central package management change.

Reply:
With that said, I guess we can merge the whole since it's better not to keep this VersionOverride exception.

To be tested with a TeamCity build.

Reply:
To be clear, that particular change was the first one I made to try and fix the build system, it wasn't enough for my case. Probably because I built it once before making any changes, leading to a stale target or something breaking further builds.